services/api/config/application.yml.example

```diff
@@ -18,25 +18,35 @@ production:
   secret_token: ~
   blob_signing_key: ~
   sso_app_secret: ~
-  sso_app_id: ~
-  sso_provider_url: ~
-  workbench_address: ~
-  websocket_address: ~
+  sso_app_id: arvados-server
+  sso_provider_url: https://uuid_prefix.your.domain
+  workbench_address: https://workbench.uuid_prefix.your.domain
+  websocket_address: wss://ws.uuid_prefix.your.domain/websocket
   #git_repositories_dir: ~
   #git_internal_dir: ~
+  #git_repo_ssh_base: "git@git.uuid_prefix.your.domain:"
+  #git_repo_http_base: https://git.uuid_prefix.your.domain/
+  #auto_admin_first_user: false
+  #crunch_job_wrapper: :slurm_immediate
+  #sso_insecure: false
```

Comment:
The real defaults (and docs) for these configs are already in `application.default.yml` -- we should avoid having an independent list of defaults here. (I'm not sure what those existing `git_*` comments are doing there but presumably we should remove them.)